protos/ensembl/production/metadata/grpc/ensembl_metadata.proto

```diff
@@ -216,6 +222,29 @@ message GenomeSequence {
   bool chromosomal = 5;
 }
 
+/*
+Metadata about the sequences that comprise a genome's assembly.
+ */
+message GenomeAssemblySequence {
+  string name = 1;
+  int32 rank = 2;
+  string md5 = 3;
+  uint32 length = 4;
```

Comment:
Max value of uint32 is ~4.29 billion. Is this going to be enough for whatever the longest chromosome is? There's a bit of a discussion [here](https://www.biostars.org/p/12560/), but might be safer to ask gene builders. Or switch to 64-bit integers.

Reply:
I'll take a look, thank you for letting me know @azangru